assets/app/scripts/services/logLinks.js

```diff
@@ -50,10 +50,28 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // broken up for readability:
+      var template = new URITemplate([
+        "/#/discover?",
+        "_g=(",
+          "time:(from:now-7y%2Fy,mode:relative,to:now)",
+        ")",
+        "&_a=(",
+          "columns:!(_source),",
+          "index:'{namespace}.*',",
+          "query:(",
+            "query_string:(",
+              "analyze_wildcard:!t,",
+              "query:'kubernetes_pod_name: {podname} %26%26 kubernetes_namespace_name: {namespace}'",
+            ")",
+          "),",
+          "sort:!(time,desc)",
+        ")",
+        "&console_back_link={backlink}"
```

Comment:
the back link and container name need to be like: https://gist.github.com/jcantrill/247f13fbc2324188a2e3#file-gistfile1-txt-L12   specifically with the '#' and joined with an '&'